libraries/Filter/LowPassFilter.cpp

```diff
@@ -59,9 +40,9 @@ T DigitalLPF<T>::apply(const T &sample) {
 template <class T>
 void DigitalLPF<T>::compute_alpha(float sample_freq, float cutoff_freq) {
     if (sample_freq <= 0) {
-        alpha = 1;
+        _alpha = 1;
     } else {
-        alpha = calc_lowpass_alpha_dt(1.0/sample_freq, cutoff_freq);
+        _alpha = calc_lowpass_alpha_dt(1.0/sample_freq, cutoff_freq);
```

Comment:
note that we are already using the `calc_lowpass_alpha_dt` function in this case.